resources/user-guides/attach-data.md

```diff
@@ -114,7 +114,7 @@ After you complete your configuration, verify that your rule looks like the foll
 
 {% tab title="Target" %}
 
-### How to attach additional profile parameters to Target retrieve location events
+### How to attach additional Target request objects to Target retrieve location events
```

Comment:
Please reword to:
"Attaching additional Target request objects to Target to retrieve location events"